src/Components/Web/src/WebEventDescriptor.cs

```diff
@@ -18,11 +18,6 @@ public sealed class WebEventDescriptor
         // This class represents the first half of the parsing process.
         // It's public only because it's part of the signature of a [JSInvokable] method.
 
-        /// <summary>
-        /// For framework use only.
-        /// </summary>
-        public int BrowserRendererId { get; set; }
```

Comment:
This is gone because the `BrowserRendererId` used to be how WebAssembly would distinguish events for different renderers. That's no longer needed because there's now a separate DotNetObjectReference representing the interop methods for each renderer.